css/index.css

```diff
@@ -20,20 +38,41 @@ ul li {
   text-align: justify;
 }
 
+div.experienceList {
+  display: flex;
+  flex-direction: column;
+}
+
+div.experienceList>.experienceItem {
```

Comment:
I think the assignment was asking to make this into a grid (ie: a table with rows and columns). Right now it still seems to look like a list.

Reply:
I will work on that, I have not used the grid because all lessons were about flexbox. I will definitely bring more changes to my portfolio.